src/services/pipeline.py

```
@@ -146,3 +147,59 @@ def pixel_art_person_cartoon(
         canvas.paste(styled_masked, (x0, y0), styled_masked)
 
     return canvas
+
+
+def pixel_art_person_anime(
+    image: Image.Image,
+    mask: np.ndarray,
+    stylizer: "AnimeStylizer",
```

Ruff (0.15.4)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)